### PR TITLE
Change admin-bundle dependency to ~2.0@dev

### PR DESCRIPTION
**Pull request by szymach**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,10 +12,10 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "fsi/admin-bundle" : "~1.1@dev",
-        "symfony/framework-bundle": "~2.3",
```
Comment (norzechowicz) [inline composer.json]: because of that there is no sense in keeping lowest dependencies support in admin-bundle.

Comment (szymach) [inline composer.json]: what do you mean?

Comment (norzechowicz) [inline composer.json]: you just forced everybody who wants to use security bundle with admin bundle to use symfony version greater than 2.8. Which means projects created on 2.3 will can't be updated now - its huge BC Breake for them.

Comment (chives) [inline composer.json]: there is no need for changing symfony dependencies

**Comment by chives** [comment]
We should also update dev-master's branch alias in this bundle to 2.0.x-dev

**Comment by norzechowicz** [comment]
And add env on travis with composer --prefer-lowest flag.

**Comment by szymach** [comment]
I'm closing this PR, because there's no point adding this twice to `master` and `remove-fosub` branches. The updates will go to `master` and then we'll simply rebase `remove-fosub`.

**Pull request closed without merging**